app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -183,7 +183,6 @@ class AttendeeViewModel(
                 if (attendees.size == totalAttendee) {
                     loadTicketsAndCreateOrder()
                     mutableIsAttendeeCreated.value = true
-                    mutableMessage.value = "Attendees created successfully!"
```

Comment:
Why?

Reply:
Why would the user want to know that attendees were created successfully. Multiple snackbars don't look good. Instead f showing snackbar at this stage, I show snackbar when order created. But haven't removed error message snackbars, so if attendees are not created successfully, user snackbar saying same will show up.

Reply:
`Why would the user want to know that attendees were created successfully`

Why wouldn't they?

Change it so that it is only shown once

Reply:
okay

Reply:
done.